libs/genai/langchain_google_genai/chat_models.py

```diff
@@ -964,12 +964,17 @@ def async_client(self) -> v1betaGenerativeServiceAsyncClient:
         # this check ensures that async client is only initialized
         # within an asyncio event loop to avoid the error
         if not self.async_client_running and _is_event_loop_running():
+            # async clients don't support "rest" transport
+            # https://github.com/googleapis/gapic-generator-python/issues/1962
+            transport = self.transport
+            if transport == "rest":
+                transport = "grpc_asyncio"
```

Comment:
ok, I see now. It might be a problem who're using a http proxy that doesn't allow grpc.

I was thinking we should just use sync client in that case, but that's not how we're doing things currently. we can fix it later then if smbd complains.